server/middlewares/authentication.js

```diff
@@ -0,0 +1,25 @@
+import JWTHelper from '../helpers/JWTHelper';
+/**
+ * @description this class handles user authentication
+ */
+class Authentication{
+
+  /**
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns {function} next
+   */
+  static authenticateUser(req, res, next) {
+    try {
```

Comment:
Lovely work here. But could you please explain why you have a `try..catch` block

Reply:
An error is being thrown from the JWT.verifyToken method.